Dockerfile

```
@@ -1,7 +1,7 @@
-ARG APP_INSIGHTS_AGENT_VERSION=2.6.1
-FROM hmctspublic.azurecr.io/base/java:openjdk-11-distroless-1.4
+ARG APP_INSIGHTS_AGENT_VERSION=3.2.6
+FROM hmctspublic.azurecr.io/base/java:openjdk-17-distroless-1.5.2
```

wherever you copied this from should be updated as well
```suggestion
FROM hmctspublic.azurecr.io/base/java:17-distroless
```